.github/workflows/cli-release.yml

```diff
@@ -0,0 +1,83 @@
+name: CLI Release
+
+on:
+  workflow_dispatch:
+    inputs:
+      draft-release:
+        description: 'Draft release?'
+        required: false
+        type: boolean
+        default: true
+      prerelease:
+        description: 'Prerelease?'
+        required: false
+        type: boolean
+        default: true
+      npm-version:
+        description: 'NPM version bump?'
+        required: false
+        type: choice
+        options: 
+          - major
+          - minor
+          - patch
+        default: 'patch'
+
+  # schedule:
+  #   - cron: '0 12 * * 2'  # Runs every Tuesday at 12:00 PM (noon) ET
+permissions:
+  contents: write
+
+jobs:
+  cli-release:
+    runs-on: ubuntu-latest
+    steps:      
+      - name: Check out code
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+
+      - name: Get node version
+        id: get_node_version
+        run: |
+          echo "NVMRC=$(cat .nvmrc)" >> $GITHUB_OUTPUT
+
+      - name: Set up Node.js
+        uses: actions/setup-node@v3
+        with:
+          node-version: "${{ steps.get_node_version.outputs.NVMRC }}"
+
+      - name: Install dependencies
+        run: yarn install
+        
+      - name: Build
+        run: yarn build
+
+      - name: Bump CLI version
+        run: npm version ${{ inputs.npm-version }} --force
+
+      - name: Get latest tag
+        run: echo "LATEST_TAG=$(git describe --abbrev=0 --tags)" >> $GITHUB_ENV
+          
+      - name: Commit and push tag to main branch
+        run: |
```

Comment:
I might be wrong here, but I think you do need the email and name configured in git as well to push git changes - like you did already on your test fork.
I did this, but the way you tested with is probably just fine too:
```
git config --global user.email "github-tracker-bot@taplytics.com"
git config --global user.name "DevCycle Automation"
```

Reply:
I'm not sure too, since @JamieSinn commented earlier said the github user is auto configured🤔. I can add this first as I recognize it's from taplytics?🤔

Reply:
It may be needed after all - since we're bypassing and pushing to main

Reply:
ok, thanks both! I'll make an update